concrete-commons/src/key_kinds.rs

```diff
@@ -0,0 +1,28 @@
+//! This module contains types to manage the different kinds of secret keys.
+
+/// This type is a marker for keys using binary elements as scalar.
+pub struct BinaryKeyKind;
+/// This type is a marker for keys using ternary elements as scalar.
+pub struct TernaryKeyKind;
+/// This type is a marker for keys using normaly sampled elements as scalar.
+pub struct GaussianKeyKind;
+/// This type is a marker for keys using uniformly sampled elements as scalar.
+pub struct UniformKeyKind;
```

Comment:
I didn't know this existed? I thought we could only have uniform binary, uniform ternary or gaussian?